src/components/card.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { product } from "../scripts/interfaces";
+import { Link } from "react-router-dom";
+
+const Card = (props: {
```

Comment:
Add an interface for these props, please and you can use destructuring `{addToCart, cart, list, listSecond}` instead of `props`.
Think about naming. list and secondList it is not clear. Maybe better `productList` or something like that.